nupic/research/frameworks/dendrites/routing/utils.py

```diff
@@ -169,7 +184,14 @@ def train_dendrite_model(
             )
 
 
-def evaluate_dendrite_model(model, loader, device, criterion, concat=False):
+def evaluate_dendrite_model(
+        model,
+        loader,
+        device,
```

Comment:
Indents here for parameters should be 4 spaces, not 8. However, feel free to put them all on the same line.